package.json

```diff
@@ -52,10 +52,10 @@
     "imagekit-javascript": "^3.0.2"
   },
   "engines": {
-    "node": ">=18.17.0"
+    "node": ">=12"
   },
   "peerDependencies": {
-    "next": "^12 || ^13 || ^14",
+    "next": "^12 || ^13 || ^14 || ^15",
     "react": "^16.13.1 || ^17.0.0 || ^18.0.0",
     "react-dom": "^16.13.1 || ^17.0.0 || ^18.0.0"
```

Comment:
add react peer dependencies for react 19 also, then we should be good to go

Reply:
done